apps/src/music/views/MusicView.jsx

```diff
@@ -390,13 +390,6 @@ class UnconnectedMusicView extends React.Component {
     // maximum possible content size, requiring no dynamic
     // resizing or user scrolling.  We did this for the dynamic
     // instructions in AI Lab.
-    const progression = this.progressManager.getProgression();
-
-    const progressState = this.progressManager.getCurrentState();
-    const currentPanel = progressState.step;
-    const message = progressState.message;
-    const satisfied = progressState.satisfied;
```

Comment:
@breville curious how you feel about this. Like I mentioned in the description, i mostly just wanted to reduce and eventually remove the usages of the `updateNumber` mechanism we've had since the early days, now that we can use redux instead. The advantage here is that components update automatically, so MusicView doesn't need to keep track of updating them manually. MusicView also doesn't need to re-render (even if the virtual DOM outside of Instructions remains the same) when a progress update happens.

However, as per your comment, now the Instructions component needs to know a little bit more about the structure of progress data. I don't think it will actually impact rendering logic in any way (instructions can still size themselves without needing dynamic resizing, as they'll re-render anytime any of the progress state changes), but it does make them logically less progress-agonstic.

Reply:
The current instructions already needed to know a bit about progress data, so I think it's alright for now for them to know some more!